ci/pipeline.yml

```
@@ -1,383 +1,384 @@
 ---
 jobs:
-- name: compile-assets
-  serial: true
-  plan:
-  - in_parallel:
-    - get: stratos-source
-      trigger: true
-      params:
-        include_source_tarball: true
-    - get: config
-      trigger: true
-  - task: precompile
-    config:
-      platform: linux
-      image_resource:
-        type: docker-image
-        source:
-          repository: node
-          tag: 16
-      inputs:
-      - name: stratos-source
-      - name: config
-      outputs:
-      - name: pre-compiled
-      params:
-        NG_CLI_ANALYTICS: "false"
-      caches:
-        - path: .npm
-        - path: stratos-source/node_modules
-      run:
-        path: sh
-        args:
-        - -exc
-        - |
-          cd stratos-source
+  - name: set-self
+    plan:
+      - get: config
+        trigger: true
+      - set_pipeline: self
+        file: cg-deploy-stratos/ci/pipeline.yml
```

```suggestion
        file: config/ci/pipeline.yml
```

Sorry, the directory is named after the resource name, these'll need to be updated too. (I couldn't figure out multi-line suggestions, thanks GitHub)

---

Thanks! I have these done now.